test/unit/repository/dnf_test.py

```diff
@@ -33,21 +33,21 @@ def setup(self, mock_path, mock_config, mock_temp):
                 ]
             )
 
-        assert runtime_dnf_config.set.call_args_list == [
-            call('main', 'cachedir', '/shared-dir/dnf/cache'),
-            call('main', 'reposdir', '/shared-dir/dnf/repos'),
-            call('main', 'varsdir', '/shared-dir/dnf/vars'),
-            call('main', 'pluginconfpath', '/shared-dir/dnf/pluginconf'),
-            call('main', 'keepcache', '1'),
-            call('main', 'debuglevel', '2'),
-            call('main', 'best', '1'),
-            call('main', 'obsoletes', '1'),
-            call('main', 'plugins', '1'),
-            call('main', 'gpgcheck', '0'),
-            call('main', 'tsflags', 'nodocs'),
-            call('main', 'arch', 'x86_64'),
-            call('main', 'ignorearch', '1'),
-            call('main', 'enabled', '1')
+        assert runtime_dnf_config.__setitem__.call_args_list == [
+            call('main', {
+                'cachedir': '/shared-dir/dnf/cache',
+                'reposdir': '/shared-dir/dnf/repos',
+                'varsdir': '/shared-dir/dnf/vars',
+                'pluginconfpath': '/shared-dir/dnf/pluginconf',
+                'keepcache': '1',
+                'debuglevel': '2',
+                'best': '1',
+                'obsoletes': '1',
+                'plugins': '1',
+                'gpgcheck': '0'
+            }),
+            call('main', {'enabled': '1'}),
+            call('main', {'enabled': '0'})
```

Comment:
This doesn't seem right? I assume this is supposed to be the plugin config?

Reply:
I'd say this part just checks what has been written to files. so it hightly depends on the order. And it was like this even before.